libs/media-player/src/audio-annotate.cy.tsx

```diff
@@ -0,0 +1,234 @@
+                it('should mark an out-point', () => {
+                    cy.wait(2000);
+
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.wait(2500);
+
+                    cy.getByDataAttribute('out-point-marker-button').click();
+
+                    cy.getByDataAttribute('time-range-selected').should('exist');
+
+                    // If I remove this cy.wait() the test suite fails at the first click
+                    cy.wait(2000);
+                });
+            });
+
+            describe(`when the mark in-point button is clicked and the user scrubs the playhead back before the in-point, clicking the mark out-point button`, () => {
+                it('should fail with an error', () => {
+                    cy.wait(4000);
+
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.get('audio').then(([audioElement]) => {
+                        audioElement.currentTime = 1.0;
+                    });
+
+                    cy.getByDataAttribute('out-point-marker-button').click();
+
+                    cy.getByDataAttribute('time-range-selected').should('not.exist');
+
+                    cy.getByDataAttribute('audio-error-message').should('exist');
+
+                    // If I remove this cy.wait() the test suite fails at the first click
+                    cy.wait(2000);
+                });
+            });
+
+            describe(`after an in-point is selected and the clear range button is clicked`, () => {
+                it('should clear the selected in-point', () => {
+                    cy.wait(4000);
+
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.wait(500);
+
+                    cy.getByDataAttribute('clear-selected-time-range-button').click();
+
+                    cy.getByDataAttribute('time-range-selected').should('not.exist');
+
+                    cy.getByDataAttribute('out-point-marker-button').should('be.disabled');
+                });
+            });
+
+            describe(`after a time range is selected and the clear range button is clicked`, () => {
+                it('should clear the range selection', () => {
+                    cy.wait(2000);
+
+                    cy.getByDataAttribute('in-point-marker-button').click();
+
+                    cy.wait(2500);
+
+                    cy.getByDataAttribute('out-point-marker-button').click();
+
+                    cy.getByDataAttribute('time-range-selected').should('exist');
+
+                    cy.wait(1000);
+
+                    cy.getByDataAttribute('clear-selected-time-range-button').click();
+
+                    cy.getByDataAttribute('time-range-selected').should('not.exist');
+                });
+            });
+
+            describe(`after a time range is selected with hotkeys and clear range is invoked with "c"`, () => {
```

Comment:
If you offer hotkeys as well as a button, you need to run every test condition once with the buttons and once with the hotkeys.